.travis.yml

```diff
@@ -27,12 +31,17 @@ addons:
       - libstdc++6:i386
       - python
       - python-pip
-  
+
+
 install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $NODE_VERSION
```

Comment:
if I understand how this works correctly, they're looking for the most recent tag, so they will need at least the history up to that point, so I'm not sure a bare clone will be acceptable.